traverser/comments/comments.go

```diff
@@ -42,14 +43,19 @@ func (t *Traverser) TraversePath(path string) error {
 		for _, b := range line {
 			curr = next
 			next = b
-			t.handleStateChange(filename, line, linecnt, prev, curr, next)
-
+			err := t.handleStateChange(filename, line, linecnt, prev, curr, next)
+			if err != nil {
+				logger.Info("Error Occured during handleStateChange of file:" + filename + " at line: " + line)
+			}
 			prev = curr
 		}
 
 		curr = next
 		next = 0
-		t.handleStateChange(filename, line, linecnt, prev, curr, next)
+		err := t.handleStateChange(filename, line, linecnt, prev, curr, next)
+		if err != nil {
```

Comment:
These errors shouldn't be handled here as they are handled via the last `return t.callbackErr`.

Does the linter allow ignoring the error via `_ = t.handleStateChange(...)`?

Alternatively, you can change the signature of the function to not return an `error` and update it accordingly

Reply:
The linter does allow `_ = t.handleStageChange(...)` so I updated the code.